packages/types-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/types-core",
-  "version": "0.0.0-2",
+  "version": "0.0.0",
```

Comment:
I wonder do we need a changelog for init release?

Reply:
I don't think so, there needs to be a reference point from what to base changes off.